index.html

```diff
@@ -15,17 +14,24 @@
       src="https://kit.fontawesome.com/04970764d6.js"
       crossorigin="anonymous"
     ></script>
+    <link rel="stylesheet" media="(min-width: 992px)" href="media.css">
+    <link rel="stylesheet" media="(max-width: 992px)" href="style.css">
   </head>
   <body>
```

Comment:
Kindly make sure that your mobile design doesn't get affected by your new desktop design. 

Your mobile design 
![image](https://user-images.githubusercontent.com/15807343/150651923-f81d509c-dd8c-4458-bb78-76c67039faf1.png)

There are some elements that are causing this issue and the reason for this is the hard-coded widths in your CSS. If horizontally, you have previous elements that are already occupying space and you sum your hard-coded width with a new element or previous margins, the result in some cases is bigger than the actual screen size which causes the problem😥

Please make the necessary changes so that your design works properly with mobile design.
To make your life easier, you could try making a media query specifically for mobile size until you figure out where is the problem 🧐